evm-contracts/src/v0.8/dev/Anchors/UniswapAnchoredView.sol

```diff
@@ -0,0 +1,388 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./UniswapConfig.sol";
+import "./UniswapV2OracleLibrary.sol";
+import "../ConfirmedOwner.sol";
+import "../../interfaces/AggregatorValidatorInterface.sol";
+
+struct Observation {
```

Comment:
I am wondering whether we could improve gas efficiency by packing this into a single slot and using a uint32 for the timestamp. Thoughts?